artifactory/services/utils/properties.go

```diff
@@ -9,92 +9,142 @@ import (
 	"strings"
 )
 
-type PropertyParseOptions int
-
-const (
-	// Parsing properties
-	SplitCommas PropertyParseOptions = iota
-	JoinCommas
-)
+const PropsSeparator = ";"
+const ValuesSeparator = ","
```

Comment:
It looks like the above two consts can be made private.
Also, you can put them both in one ```const ( )``` phrase.